go.mod

```diff
@@ -1,75 +1,74 @@
 module github.com/crossplane-contrib/provider-helm
 
-go 1.22.0
+go 1.22.1
 
-toolchain go1.22.1
+replace github.com/crossplane-contrib/provider-kubernetes => github.com/turkenh/provider-kubernetes v0.0.0-20240523170107-bea981f8819d
```

Comment:
TODO: remove this when https://github.com/crossplane-contrib/provider-kubernetes/pull/251 merged.